### PR TITLE
Merge 0.4 into 0.5

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary

Resolves merge conflicts due to recent changes in 0.4

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -28,10 +28,28 @@ Changes are ordered reverse-chronologically.
  - KOLIBRI_LISTEN_PORT environment variable for specifying a default for the --port option #1724
 
 
+0.4.6
+-----
+
+ - Fix various exam and progress tracking issues
+ - Add automatic sign-out when browser is closed
+ - Fix search issue
+ - Learner UI updates
```
Comment (indirectlylit) [inline CHANGELOG.rst:37]: Maybe add 'updated hindi translations' here? We can cherry-pick the commit back to the 0.4 branch

**Comment by radinamatic** [comment]
Ubuntu looks OK too, good to go! :+1: 

fullscreen_19_10_17_18_28

File: CHANGELOG.rst (hunk 1 above)
Comment (rtibbles) [reply CHANGELOG.rst:37]: OK, can do. Will need to update translations on 0.5 as well.

Comment (rtibbles) [reply CHANGELOG.rst:37]: Will do in the same PR.

**Pull request merged**